packages/components/src/components/form-submit-error-message/form-submit-error-message.tsx

```diff
@@ -1,20 +1,40 @@
-import React from 'react';
 import classNames from 'classnames';
 import Icon from '../icon/icon';
 import Text from '../text';
+import { Localize } from '@deriv/translations';
+import OpenLiveChatLink from '../open-livechat-link';
 
 type TFormSubmitErrorMessage = {
     className?: string;
     message: string;
+    error_code?: string;
 };
 
-const FormSubmitErrorMessage = ({ className, message }: TFormSubmitErrorMessage) => (
-    <div className={classNames('dc-form-submit-error-message', className)}>
-        <Icon icon='IcAlertDanger' data_testid='form_submit_error' />
-        <Text as='p' size='xxs' weight='bold' color='prominent'>
-            {message}
-        </Text>
-    </div>
-);
+const FormSubmitErrorMessage = ({ className, message, error_code }: TFormSubmitErrorMessage) => {
+    const getErrorMessage = () => {
+        switch (error_code) {
+            case 'PhoneNumberTaken':
+                return (
+                    <Text as='p' size='xxs' className='dc-form-submit-error-message__phone-number-taken'>
+                        <Localize
+                            i18n_default_text='Number already exists in our system. Enter a new one or contact us via <0></0> for help'
+                            components={[<OpenLiveChatLink text_size='xxs' key={0} />]}
+                        />
+                    </Text>
+                );
+            default:
+                <Text as='p' size='xxs' weight='bold' color='prominent'>
+                    {message}
+                </Text>;
+        }
+    };
+
+    return (
+        <div className={classNames('dc-form-submit-error-message', className)}>
+            <Icon icon='IcAlertDanger' data_testid='form_submit_error' />
```

Comment:
can we use Quill icon here?

Reply:
@Dhruv-deriv this is legacy design 🙏 there's no @deriv-com/quill-ui install in core component package.

Reply:
Hi @suisin-deriv asked a question in DM :)

Reply:
LGTM @suisin-deriv 👍